nav2_dijkstra_planner/src/dijkstra_planner.cpp

```diff
@@ -145,14 +148,14 @@ DijkstraPlanner::makePlan(
   double wx = start.position.x;
   double wy = start.position.y;
 
-  RCLCPP_INFO(get_logger(), "DijkstraPlanner::makePlan: from %.2f,%.2f to %.2f,%.2f",
+  RCLCPP_INFO(get_logger(), "DijkstraPlanner: Making plan from (%.2f,%.2f) to (%.2f,%.2f)",
     start.position.x, start.position.y, goal.position.x, goal.position.y);
 
   unsigned int mx, my;
   if (!worldToMap(wx, wy, mx, my)) {
     RCLCPP_WARN(
       get_logger(),
-      "DijkstraPlanner::makePlan: The robot's start position is off the global costmap."
+      "DijkstraPlanner: Cannot create a plan: the robots start position is off the global costmap."
```

Comment:
```suggestion
      "DijkstraPlanner: Cannot create a plan: the robot's start position is off the global costmap."
```
I believe the apostrophe was correct here since it's a possessive instead of a plural.